### PR TITLE
[14.0][MIG] purchase_deposit

### PR DESCRIPTION
**Pull request by kayue**
Standard migration, ready for review.

Would be great if someone can review the ir.model.access.csv.

### CODE REVIEW

**Comment by kittiu** [comment]
@kayue 
The travis still red, you may need to run pre-commit again.
By the way, for the MIG work, you can follow the migration instruction here, https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#technical-method-to-migrate-a-module-from-130-to-140-branch
If you done it right, you should have 2 commits,
* [IMP] $MODULE: black, isort, prettier
* [MIG] $MODULE: Migration to 14.0

**Comment by kayue** [comment]
Not sure how to run the `pre-commit run -a` command. pre-commit isn't a tool I installed

**Comment by kittiu** [comment]
> Not sure how to run the `pre-commit run -a` command. pre-commit isn't a tool I installed

It should be pretty simple, `pip3 install pre-commit`
Please try out it is very useful tool.

**Comment by kayue** [comment]
@kittiu Thanks, didn't even know it is a tool I can install at the first placed. 

Updated the PR.

**Comment by kittiu** [comment]
Please run `pre-commit run -a` again. Travis is still red.

**Comment by kittiu** [comment]
Looks like error is from purchase_work_acceptance.
https://travis-ci.com/github/OCA/purchase-workflow/jobs/457094108#L1587

@SurachetS can you look at this please?

**Comment by kittiu** [comment]
cc @SurachetS @PrapassornS

File: purchase_deposit/models/purchase.py [hunk 1]
```diff
@@ -27,8 +27,8 @@ class PurchaseOrderLine(models.Model):
         " order. They are not copied when duplicating a purchase order.",
     )
 
-    def _prepare_account_move_line(self, move):
-        res = super(PurchaseOrderLine, self)._prepare_account_move_line(move)
+    def _prepare_account_move_line(self):
```
Comment (kittiu) [inline purchase_deposit/models/purchase.py:30]: @kayue I think you can not remove param `move` from the function like this. It was there in the core odoo.
https://github.com/odoo/odoo/blob/14.0/addons/purchase/models/purchase.py#L1092

Comment (kittiu) [reply purchase_deposit/models/purchase.py:30]: Can you fix this and also rebase your branch, it was quite old. :)

**Comment by kittiu** [comment]
@kayue are you planning to continue with this PR?

**Comment by kayue** [comment]
@kittiu sorry no plan to continue this, feel free to take over it. thanks.

**Comment by kittiu** [comment]
> @kittiu sorry no plan to continue this, feel free to take over it. thanks.

OK, replaced by https://github.com/OCA/purchase-workflow/pull/1178

**Pull request closed without merging**